provisioner/terraform/resources.go

```diff
@@ -385,6 +385,7 @@ func ConvertState(modules []*tfjson.StateModule, rawGraph string) (*State, error
 	resourceIcon := map[string]string{}
 	resourceCost := map[string]int32{}
 
+	metadataTargetLabels := map[string]struct{}{}
```

Comment:
Commentary: Here I think a `map[string]bool` would be ideal, simplifies the check below where we can simply do `if metadataTargetLabels[targetLabel]`. The space savings from using `struct{}` should be negligible.

No change required, just an observation.

Reply:
Fixed